spec/features/saml/loa3_sso_spec.rb

```diff
@@ -44,29 +44,56 @@
     end
   end
 
-  context 'canceling verification', js: true do
-    it 'returns user to recovery code page if they sign up via loa3' do
-      user = create(:user, phone: '1 (111) 111-1111', recovery_code: nil)
-      sign_in_with_warden(user)
-      loa3_sp_session
+  context 'canceling verification' do
+    context 'with js', js: true do
+      it 'returns user to recovery code page if they sign up via loa3' do
+        user = create(:user, phone: '1 (111) 111-1111', recovery_code: nil)
+        sign_in_with_warden(user)
+        loa3_sp_session
 
-      visit verify_path
+        visit verify_path
 
-      click_on t('links.cancel')
-      click_on t('idv.buttons.cancel')
+        click_on t('links.cancel')
+        click_on t('idv.buttons.cancel')
 
-      expect(current_path).to eq(manage_recovery_code_path)
+        expect(current_path).to eq(manage_recovery_code_path)
+      end
+
+      it 'returns user to profile page if they have previously signed up' do
+        sign_in_and_2fa_user
+        loa3_sp_session
+
+        visit verify_path
+
+        click_on t('links.cancel')
+        click_on t('idv.buttons.cancel')
+        expect(current_url).to match(/profile/)
```

Comment:
no matter what the entire URL is, `expect(current_path).to eq profile_path` should work